prisma/schema.prisma

```diff
@@ -96,6 +109,10 @@ model User {
   endTime            DateTime? @map("end_time") @db.Time(0)
   favoritedBy        User[]    @relation("Favorites")
   favorites          User[]    @relation("Favorites")
+  sentInvitations    Invitation[]  @relation("sentInvitations")
+  receivedInvitations Invitation[] @relation("receivedInvitations")
+  connectedBy        User[] @relation("Connections")
```

Comment:
Should users be allowed to connect with multiple people? Can the attribute just be some ``carpoolGroup`` instead which is a list of ``User``?

Reply:
Yes Luke is right here. @owensimpson connected should be just one person. I think the carpoolGroup model is a potential idea, let's talk about it more during standup